### ISSUE TITLE
redis.log call in Lua script fails

### ISSUE THREAD

**Issue by upcFrost** [jamesls/fakeredis#265]
Hi,

Lua scripts fail for me if they use the `redis.log` directive. 

Example:
```
def test_redis_lua(self):
        lua = """redis.log(redis.LOG_WARNING, "foo bar")"""
        script = self.redis.register_script(lua)
        script()
```

Error:
```
.......
except LuaError as ex:
>           raise redis.ResponseError(str(ex))
E           redis.exceptions.ResponseError: [string "<python>"]:1: attempt to call a nil value (field 'log')
E           stack traceback:
E           	[string "<python>"]:1: in main chunk

../../venv/3/lib/python3.7/site-packages/fakeredis/_server.py:2352: ResponseError
```

I'm using fakeredis-1.2.1.

Adding a very basic stub like `redis.log = function(lvl, msg) return nil end` to `_server.py:2333` works for me, but ofc it'd be nice to log the messages

**Comment by bmerry** [comment]
I'm afraid I don't use Lua myself, so I rely on PRs from users to extend the Lua functionality. If you have an idea of how log messages should be mapped to the Python logging system then please feel free to submit a PR.

**Comment by upcFrost** [comment]
Ok, I'll try to make one 👌

**Comment by bmerry** [comment]
Closed via #266.
